### PR TITLE
feat: add Amazon DCV Windows module

### LINKED ISSUE #344: Amazon DCV Windows Integration
High-Performance Windows Remote Desktop Client in the web browser & via desktop.

### PR DESCRIPTION
**Pull request by matifali**
Adds Amazon DCV for Windows workspaces. 

Closes #344 along with coder/coder#15610

### CODE REVIEW

File: amazon-dcv-windows/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+icon: ../.icons/dcv.svg
+maintainer_github: coder
+partner_github: aws
+verified: true
+tags: [windows, dcv, web, desktop]
+---
+
+# Amazon DCV Windows
+
+Amazon DCV is high performance remote display protocol that provides a secure way to deliver remote desktop and application streaming from any cloud or data center to any device, over varying network conditions.
```
Comment (bpmct) [inline amazon-dcv-windows/README.md:13]: Can we add a nice screenshot or GIF for marketing purposes?

File: amazon-dcv-windows/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+display_name: Amazon DCV Windows
+description: Amazon DCV Server and Web Client for Windows
+icon: ../.icons/dcv.svg
+maintainer_github: coder
+partner_github: aws
+verified: true
+tags: [windows, dcv, web, desktop]
+---
+
+# Amazon DCV Windows
```
Comment (bpmct) [inline amazon-dcv-windows/README.md:11]: For non AWS workspaces, does this work OK?

**Review by bpmct — APPROVED** [review]

File: amazon-dcv-windows/README.md (hunk 2 above)
Comment (matifali) [reply amazon-dcv-windows/README.md:11]: Yes. I tested it on GCP, and it works. You will see a notification about getting the license, though.

**Comment by bpmct** [comment]
Are we good to merge?

**Pull request merged**